src/PRASBase/assets.jl

```diff
@@ -88,7 +88,7 @@ struct Storages{N,L,T<:Period,P<:PowerUnit,E<:EnergyUnit} <: AbstractAssets{N,L,
     μ::Matrix{Float64}
 
     function Storages{N,L,T,P,E}(
-        names::Vector{String}, categories::Vector{String},
+        names::Vector{AbstractString}, categories::Vector{String},
```

Comment:
I think this should be `Vector{<:AbstractString}` (not `Vector{AbstractString}`)? And we'll want to change the type of `categories` as well. Same applies for `GeneratorStorages` and `Lines`.

Reply:
Oops you're right, let me fix that.